service/src/main/java/bio/terra/workspace/db/ApplicationDao.java

```diff
@@ -342,12 +344,30 @@ public void updateApplication(WsmApplication app) {
 
     var params =
         new MapSqlParameterSource()
-            .addValue("application_id", app.getApplicationId().toString())
+            .addValue("application_id", app.getApplicationId())
             .addValue("display_name", app.getDisplayName())
             .addValue("description", app.getDescription())
             .addValue(SERVICE_ACCOUNT, app.getServiceAccount())
             .addValue("state", app.getState().toDb());
 
     jdbcTemplate.update(sql, params);
   }
+
+  /**
+   * Temporary method to delete the old Leo application. This is based on the understanding that
+   * there is no use of the application anywhere. No enabling in any workspace and no dependent
```

Comment:
Yeah, I just generated this UUID to make the config. Shouldn't be used anywhere.